azkaban-exec-server/src/main/java/azkaban/execapp/ExecMetrics.java

```diff
@@ -29,21 +31,50 @@
 public class ExecMetrics {
   public static final String NUM_RUNNING_FLOWS_NAME = "EXEC-NumRunningFlows";
   public static final String NUM_QUEUED_FLOWS_NAME = "EXEC-NumQueuedFlows";
-  public static final String PROJECT_DIR_CACHE_HIT_RATIO_NAME = "EXEC-ProjectDirCacheHitRatio";
-  public static final String FLOW_SETUP_TIMER_NAME = "EXEC-flow-setup-timer";
+  public static final String PROJECT_DIR_CACHE_HIT_RATIO_NAME = "project-dir-cache-hit-ratio";
+  public static final String FLOW_SETUP_TIMER_NAME = "flow-setup-timer";
+  public static final String QUEUE_WAIT_HISTOGRAM_NAME = "queue-wait-histogram";
+  public static final String FLOW_TIME_TO_START_HISTOGRAM_NAME= "flow-time-to-start-histogram";
+  public static final String FLOW_KILLING_METER_NAME = "flow-killing-meter";
+  public static final String FLOW_TIME_TO_KILL_HISTOGRAM_NAME = "flow-time-to-kill-histogram";
+  public static final String FLOW_KILLED_METER_NAME = "flow-killed-meter";
+  public static final String FLOW_SUCCESS_METER_NAME = "flow-success-meter";
+  public static final String JOB_FAIL_METER_NAME = "job-fail-meter";
+  public static final String JOB_SUCCESS_METER_NAME = "job-success-meter";
+  public static final String JOB_KILLED_METER_NAME = "job-killed-meter";
 
   private final MetricsManager metricsManager;
   private Timer flowSetupTimer;
   private final ProjectCacheHitRatio projectCacheHitRatio;
+  private Histogram queueWaitHistogram;
+  private Histogram flowTimeToStartHistogram;
+  private Meter flowKillingMeter;
```

Comment:
You're right, both of them as meters are basically the same thing with a time shift. However, if we have KILLING as a counter it is useful to know how many flow executions are in KILLING status in a point in time and set alerts based on that as executions can get stuck in this status for different reasons.
Kill is a user action, so to me the KILLED metric has more statistical value rather than being an indicator of the health of the system. 
LMKWYT

Reply:
Sounds good. Thanks for giving it a thorough consideration.